cmd/sealos/cmd/root.go

```diff
@@ -57,6 +58,8 @@ func init() {
 	_ = fs.MarkHidden("cluster-root")
 	fs.StringVar(&runtimeRootDir, "sealos-root", constants.DefaultRuntimeRootDir, "root directory for sealos actions")
 	_ = fs.MarkHidden("sealos-root")
+	// add unrelated command names that don't required buildah sdk.
+	buildah.AddUnrelatedCommandNames("cert", "docs", "exec", "gen", "scp", "version")
```

Comment:
Move this params to const ahead of this file?

Reply:
Each subcommands can call function `buildah.AddUnrelatedCommandNames` separately, so I don't see the need to make it as constant. @cuisongliu what do you suggest?